tests/run_vector_tests.sh

```diff
@@ -9,18 +9,22 @@ fi
 
 pip install -r "$1/requirements.txt"
 $1/generator.py -t uptane --signature-encoding base64 -o vectors --cjson json-subset
-$1/server.py -t uptane --signature-encoding base64 &
+if [ "$2" == "valgrind" ]; then
+  $1/server.py -t uptane --signature-encoding base64 -P 8081 &
```

Comment:
@OYTIS There'd still be the problem that only the server script would know about the port. I'd have to communicate to the C++ program as well. I'll look into a way to find an open port natively in python so I can call it directly in run_vector_tests and then pass it to both programs.